go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/mr-tron/base58 v1.2.0
 	github.com/nspcc-dev/dbft v0.0.0-20210721160347-1b03241391ac
 	github.com/nspcc-dev/go-ordered-json v0.0.0-20220111165707-25110be27d22
-	github.com/nspcc-dev/neo-go/pkg/interop v0.0.0-20220114100620-29afa8e7b512
+	github.com/nspcc-dev/neo-go/pkg/interop v0.0.0-20220118080652-4eddfdbbc652
```

Comment:
Just a question: from now on every time we're changing something inside the interop package we should also update this dependency, shouldn't we?

Reply:
Yes and sadly this should be done in 2 steps, first updating `pkg/interop` itself and then all other code.
Not a big problem for developing though, as we can `replace` it in `go.mod`.